Microsoft.Toolkit.Uwp.SampleApp/SamplePages/GridSplitter/GridSplitter.bind

```diff
@@ -85,14 +84,6 @@
       <controls:GridSplitter.RenderTransform>
         <TranslateTransform Y="-8" />
       </controls:GridSplitter.RenderTransform>
-      <controls:GridSplitter.Element>
```

Comment:
BREAKING CHANGE: When removing the reliance upon the now-defunct `GripperHoverWrapper` class, this then allowed further refactoring, to where this `Element` property behaved more or less just like the framework `Content` property, so I refactored the `Element` property out.  

- The change would be invisible to anyone just using the default `GridSplitter` settings, and would not introduce any breaking changes. 
- However, for anyone that has a custom implementation of the `GridSplitter`, this could break the customization after we remove this `Element `property.

Reply:
Yeah, if we can just use the default `Content` pattern, then that's going to be fine to remove the custom `Element` thing. :)